test/util.go

```diff
@@ -42,13 +44,55 @@ func AssertImageMatches(t *testing.T, masterFilename string, img image.Image) bo
 	expected := image.NewRGBA(raw.Bounds())
 	draw.Draw(expected, expected.Bounds(), raw, image.Pt(0, 0), draw.Src)
 
-	if !assert.Equal(t, expected, img, "Image did not match master. Actual image written to %s.", failedPath) {
+	var msg string
+	if len(msgAndArgs) > 0 {
+		msg = fmt.Sprintf(msgAndArgs[0].(string)+"\n", msgAndArgs[1:]...)
+	}
+	if !assert.Equal(t, expected, img, "%sImage did not match master. Actual image written to file://%s.", msg, failedPath) {
 		require.NoError(t, writeImage(failedPath, img))
 		return false
 	}
 	return true
 }
 
+// MoveMouse simulates a mouse movement to the given position.
+func MoveMouse(c fyne.Canvas, pos fyne.Position) {
```

Comment:
Why?

Reply:
Fair question. I just wondered if people seeing this test might lead to conversations like "how do I get moved callback in my app" and the answer is "it's in desktop extensions"

Reply:
Well, I think people who are looking at the test framework are able to use their editors search functionality ;).
Also a mouse is desktop specific, so that might be hint enough?